tests/network/samba/samba_adcli.pm

```diff
@@ -90,7 +93,7 @@ sub update_password {
                 record_soft_failure("bsc#1188575");
                 last;
             } else {
-                die("wbinfo output does not contain $substr");
+                record_soft_failure("wbinfo output does not contain $substr");
```

Comment:
use `record_soft_failure` only with ticket references, see https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/Makefile#L146

Reply:
you are right! thanks